doc/src/stdlib/linalg.md

```diff
@@ -73,6 +73,10 @@ Base.LinAlg.diagm
 Base.LinAlg.scale!
 Base.LinAlg.rank
 Base.LinAlg.norm
+Base.LinAlg.norm(::AbstractVector, ::Real)
```

Comment:
Without these, the docstrings for specific methods do not appear at all in the manual, but with these the methods appear each in a separate box, contrary to what happens e.g. for `vecnorm`. I'm probably missing something. @MichaelHatherly